latex/Research_Strategy.tex

```diff
@@ -196,9 +201,10 @@
 %SAMPL drives science in our groups and in the field
    %a. Give examples from our groups from prior SAMPLs
    %b. Examples from field in prior SAMPLs
-Blind predictive challenges -- and SAMPL in particular -- already play a key role in fostering innovation in the field, especially in the form of method development, testing, and hardening, as well as force field development. 
-Thus, this work, by continuing and extending SAMPL, will advance innovative new science. 
-It is worth briefly highlighting several historical examples, though far more are available in the SAMPL literature. 
+Blind predictive challenges---and SAMPL in particular---already play a key role in fostering innovation in the field, especially in the form of method development, evaluation, and hardening, as well as force field development. 
+By expanding SAMPL, will advance innovative new science. 
+{\color{red}[JDC: How is it innovative if it has been going on for a while? Can we point to the number of new methodology publications it has motivated, for example?]}
```

Comment:
> I don't think the innovation of this proposal is that it's going to do something fundamentally new, and I think we have to be honest about that.

I think it does help if we stress that we're going beyond what was already done, and see if we can communicate WHY it the proposed expansion of SAMPL is innovative and necessary, lest they think we should just carry on as we have without funding.